### PR TITLE
Create fibonacci.py

### PR DESCRIPTION
**Pull request by NishantMR7**
This Pull Request resolves Issue #<IssueNumber>

Description:
In this Pull Request we have done the following additions/edits for the code:

Note: Delete all the text including this!
* Make sure you replace `#<IssueNumber>` with the particular Issue that you are referencing with!
* Make sure you describe the Issue properly and ask as many people as possible to review your Pull Request so that it can be merged as soon as possible!

### CODE REVIEW

File: Fibonnaci_Search/fibonacci.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+F = [0,1,1,2,3,5,8]
+ListElements = sorted([83,24,65,123,175,57,123,243])
```
Comment (MJ10) [inline Fibonnaci_Search/fibonacci.py]: Please do not use hardcoded arrays, accept them from user input.

File: Fibonnaci_Search/fibonacci.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+X = 243
+
+p = len(F)
+k = F[p-1]
+
+if(k == 0):
+    print("K = 0")
+else:
+    while(True):
+        print("test1")
```
Comment (MJ10) [inline Fibonnaci_Search/fibonacci.py]: Remove the test prints

**Comment by MJ10** [comment]
@NishantMR7 Any updates?

**Comment by salman-bhai** [comment]
Pull Request will be closed in 10 hours. Please send required changes before that.

**Pull request closed without merging**